README.md

```diff
@@ -143,25 +233,27 @@ We run as much as we can on Heroku, because that environment puts devs in direct
 
 The remainder of our environment is hosted in AWS. We make heavy use of EC2, RDS, EBS, and S3.
 
-### Testing & monitoring
+### Testing & Monitoring
 
 We maintain quality with unit and functional testing, and we’re constantly trying to find better ways to test. We strongly prefer frameworks, like Laravel, that have unit testing built in to their core.
 
-We monitor with New Relic, StatHat, Runscope, and Ghost Inspector. We centralize alerts and alerting policies in PagerDuty.
+We [monitor](https://github.com/DoSomething/communal-docs/tree/master/Monitoring) with New Relic, StatHat, [Runscope, and Ghost Inspector](https://github.com/DoSomething/communal-docs/tree/master/Automated%20Tests). We centralize alerts and alerting policies in PagerDuty. We have an [Incident Playbook](https://github.com/DoSomething/communal-docs/tree/master/Incidents) and we document common issues we might see for a given service.
```

Comment:
Do you think it's worth mentioning here that we use an on-call rotation & including a link to the [RFC](https://github.com/DoSomething/rfcs/blob/master/007-on-call.md)?